server.js

```diff
@@ -55,208 +56,229 @@ function showHomepage(req, res) {
-  //------------------------------------------------------------------------------
-  //set the url for smithsonian API
-  let url = `https://api.si.edu/openaccess/api/v1.0/category/art_design/search?q=${artist}&api_key=${process.env.SMITHSONIAN_APIKEY}`;
-
-  //call the smithsonian's API
-  superagent.get(url)
-    .then(smithsonianData => {
-      //after every.tehn turn this into a function//
-      //create an array of artworks that we will add all the smithsonian results to
-      var allArtworks = [];
-      //narrow down the results to those where the artist name matches the search query by using .filter on the returned array.
-      //this API doesn't let you narrow the search to be by artist name only, so we have to do it manually here.
-      var rows = [];
-      if (smithsonianData.body.response.rows.length > 0) {
-        rows = smithsonianData.body.response.rows.filter(item => {
-          if (item.content.freetext.name && item.content.freetext.name.length > 0) {
-            return item.content.freetext.name[0].content.toLowerCase().indexOf(artist.toLowerCase()) > -1;
-          }
-        });
-      }
-      //now iterate on the remaining rows and add the artworks to the array we created
-      rows.forEach(item => {
-        allArtworks.push(new ArtWork(
-          item.content.descriptiveNonRepeating.data_source,
-          artist,
-          item.title,
-          //if there is online_media, then check how many items are in it. If more than 0, then set the image URL to the thumbnail of the first image. Otherwise, set this field to null so we don't render an image on the page.
-          (item.content.descriptiveNonRepeating.online_media && item.content.descriptiveNonRepeating.online_media.mediaCount > 0) ? item.content.descriptiveNonRepeating.online_media.media[0].thumbnail : null,
-          //if there are notes describing the artwork, save the first note's content.  Otherwise, set this field to null so we don't display it on the page
-          (item.content.freetext.notes && item.content.freetext.notes.length) > 0 ? (item.content.freetext.notes[0].content ? item.content.freetext.notes[0].content : null) : null,
-          'Washington D.C.'
-        ));
-      });
-      console.log(allArtworks);
-      return allArtworks;
-
-    })
-    //------------------------------------------------------------------------------
-    // Get the results for the search query from the MET's api
-    //------------------------------------------------------------------------------
-    //then, take the array of Artwork objects we created from the Smithsonian superagent call, and send it to get MET results
-    .then(data => {
-      var allArtworks = data;
-      //get the results for the search query from the MET
-      let url = `https://collectionapi.metmuseum.org/public/collection/v1/search?q=${artist}&artistOrCulture`;
-      //call the MET's API which will return a list of object IDs that match the search query.
```

Comment:
a lot of code deleted here that shouldn't have been.

Reply:
I can't quite tell what should be put back because it sorta moved stuff around, but bottom line is you can copy and paste everything that you put inside the if statement from the github staging repo on the web to make sure it's not deleting anything necessary.

Reply:
It's weird because those lines are still in my code. I think it could be from when I was writing out an sl to check for duplicates and then moved it, so really the code is still there, but lines may have shifted?

Reply:
Line 139 is now on line  142 in my code, so yeah I think was just shifted down from adding things up higher in the code.